server/py/framework/db/sqldb/db.py

```diff
@@ -7754,3 +7846,21 @@ def _validate_integer_max_value(column: Column, value: int):
             raise mlrun.errors.MLRunInvalidArgumentError(
                 f"Unsupported column type '{column.type}' for validation."
             )
+
+    def _update_model_link(self, session, mep_record: ModelEndpoint, model_path: str):
+        if model_path and mlrun.datastore.is_store_uri(model_path):
```

Comment:
You have already verified it on the caller side. You could also move the verification of store uri to the caller. 
```suggestion
        if mlrun.datastore.is_store_uri(model_path):
```